plugins/mcreator-core/features/feature_huge_red_mushroom.json

```diff
@@ -0,0 +1,37 @@
+{
+  "args0": [
+    {
+      "type": "input_value",
+      "name": "cap",
+      "check": "BlockStateProvider"
+    },
+    {
+      "type": "input_value",
+      "name": "stem",
+      "check": "BlockStateProvider"
+    },
+    {
+      "type": "field_number",
+      "name": "radius",
+      "value": 2,

```

Comment:
It may make sense to limit minimum to 0 or 1, as negative values will probably crash something, same for brown mushroom